glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
++ #### *Encryption* is a way of modifying a message, so that only authorized parties are able to interpret it
++ #### *Decryption* is a process of retrieving data from an encrypted message
++ #### *Muxing*(abbr. from multiplexing) is a method of combining multiple signals into one signal over a shared medium. Such signal can be then [demuxed](/glossary/glossary#demuxing) back into original signals
++ #### *Demuxing*(abbr. from demultiplexing) is a method of separating signals from one combined signal
++ #### Server's architecture:
+  + #### *SFU*(Selective Forwarding Unit) is a video conferencing architecture which consists of a single server, which receives incoming streams from all participants and forwards each participant's stream to all other conference participants
+  + #### *MCU*(Multipoint Control Unit) is a device responsible for connecting conference participants and controlling the stream
+  + #### *P2P*(Peer to Peer) is an architecture in which each participant is directly connected to all other participants, which eliminates the need for MCU 
+
+
+# Membrane Framework 
++ #### *Pad* is an input or output of an element. Output pads of one element are connected to input pads of another element.
++ #### *Caps*(abbr. from capabilities) define [pads](/glossary/glossary#pad) specification, allowing us to determine whether two elements are compatible with each other 
++ #### *Pipeline* is a chain of linked [elements](/glossary/glossary#element) which together accomplish some media processing task
++ #### *Buffer* is a fundamental structure in Membrane used to send data between elements
++ #### *Element* is the most basic entity responsible for processing multimedia. Each element is created to solve one problem. Elements can be divided into three categories:
+  + #### *Source* is an element with only output pads, the first element of each pipeline. It is responsible for fetching the data and transmitting it through the output pad
+  + #### *Filter* is an element with both input and output pads, which is responsible for transforming data
+  + #### *Sink* is an element with only input pads, the last element of a pipeline
++ #### Types of elements:
+  + #### *Payloader* and *Depayloader* are responsible for respectively dividing frames into packets and assembling packets back into frames
+  + #### *Encoder* and *Decoder* are responsible for [encoding](/glossary/glossary#encoding) and [decoding](/glossary/glossary#decoding)
+  + #### *Encryptor* and *Decryptor* are responsible for [encryption](/glossary/glossary#encryption) and [decryption](/glossary/glossary#decryption)
+  + #### *Muxer* and *Demuxer* are responsible for [muxing](/glossary/glossary#muxing) and [demuxing](/glossary/glossary#demuxing)
+  + *mixer* 
+  + #### *Jitter buffer / Ordering buffer* is an area of memory, which is used to temporarily store data incoming from network, used to arrange the data in the order in which it was sent
```

Comment:
I would stick to the convention "(...) is an element" instead of "is an area of a memory". You can also mention why do we need it (that's due to unreliability of the network that the packets can arrive out of the order)